dataflow/src/main/java/org/checkerframework/dataflow/cfg/visualize/CFGVisualizeLauncher.java

```diff
@@ -267,12 +269,35 @@ public void write(int b) throws IOException {}
         return res.getCFG();
     }
 
+    /**
+     * Write down generated String representation of the CFG.
+     *
+     * @param analysis instance of forward or backward analysis from specific dataflow test case.
+     */
+    @SuppressWarnings("CatchAndPrintStackTrace") // we want to use e.printStackTrace here.
+    public static void writeDownStringOfCFG(Analysis<?, ?, ?> analysis) {
```

Comment:
Odd name.

Reply:
Thanks, changed the name to "writeStringOfCFG"